### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by joshua-shuhan**
Add test result

### CODE REVIEW

**Comment by TrevorZzzzz** [comment]
okay okay, i am glad i am negative, doctor

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # PCR_test
 -miss or mr 郑桑， please do what every you want
+
+- The test result is negative.
```
Comment (TrevorZzzzz) [inline README.md:4]: yeahhhhhh! negative

**Review by TrevorZzzzz — COMMENTED** [review]
my life saved from negative result

File: README.md (hunk 1 above)
Comment (joshua-shuhan) [reply README.md:4]: looks good

**Comment by TrevorZzzzz** [comment]
do you mind we two add wechat together, mine number is zgh15999001100

File: README.md (hunk 1 above)
Comment (joshua-shuhan) [reply README.md:4]: here is my repo~
https://github.com/joshua-shuhan/media_computing_test

**Pull request merged**